README.md

```diff
@@ -3,3 +3,4 @@
 ## Contributors
 Michael Sherman (michaelsherman@google.com)
 
```

Comment:
There's no need to leave the empty line, though this may effectively disappear after the other merges. I'm going to approve this but make sure there's no empty line before you merge the pull request (there are 3 merges coming which may conflict with each other, we'll see).